src/xcode/ENA/ENA/Source/AppDelegate & Globals/ENATaskExecutionDelegate.swift

```diff
@@ -323,9 +323,9 @@ class TaskExecutionHandler: ENATaskExecutionDelegate {
 		})
 	}
 
-	private func executeCCLConfigurationUpdateAndBoosterNotificationsCreation(completion: @escaping () -> Void) {
-		Log.info("Checking if Booster rules need to be downloaded...", log: .vaccination)
-		healthCertificateService.checkForCCLConfigurationAndRulesUpdates(completion: completion)
+	private func executeDCCWalletInfoUpdatesAndTriggerBoosterNotificationsIfNeeded(completion: @escaping () -> Void) {
+		Log.info("[ENATaskExecutionDelegate] Checking if DCC wallet infos need to be updated and booster notifications need to be triggered...", log: .vaccination)
```

Comment:
`[ENATaskExecutionDelegate]` -> Isn't the method already logged by our logger, so why add it extra into the log statement?

Reply:
That's true. I think sometimes it's hard to filter for something like this because the file is in a separate line of the log. But in this case I just made it consistent with the functions above, that also have it.

I added a topic to our DevCamp to improve this situation :)